ddtrace/monkey.py

```diff
@@ -1,23 +1,13 @@
-"""Patch librairies to be automatically instrumented.
```

Comment:
Why is this being removed entirely?

Reply:
It should be replaced with a new docstring.